frontend/src/features/leases/detail/LeasePages/deposits/models/FormLeaseDeposit.ts

```diff
@@ -0,0 +1,57 @@
+import { fromContact, IContactSearchResult, toContact } from 'interfaces';
+import { Api_SecurityDeposit } from 'models/api/SecurityDeposit';
+import { NumberFieldValue } from 'typings/NumberFieldValue';
+
+export class FormLeaseDeposit {
```

Comment:
I think we need to talk about this, I'm starting to think that having half of our forms using classes and half using interfaces isn't such a hot idea.